controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	KubeadmControlPlaneFinalizer = "kubeadmcontrolplane.cluster.x-k8s.io"
+	KubeadmControlPlaneFinalizer = "kubeadmcontrolplane.controlplane.cluster.x-k8s.io"
```

Comment:
```suggestion
	KubeadmControlPlaneFinalizer = "kubeadm.controlplane.cluster.x-k8s.io"
```

Reply:
Hah, I did switch to that and then back again. Not sure what's right given the object is `kubeadmcontrolplane.controlplane.cluster.x-k8s.io`.

@detiber ?

Reply:
I'm good with shortening the finalizer. It would be difficult to do the same with the type because a `Kind` of `Kubeadm` lacks context for what the type does.

Reply:
Thanks. Done